components/fxa-client/Cargo.toml

```diff
@@ -26,13 +26,14 @@ thiserror = "1.0"
 anyhow = "1.0"
 sync-guid = { path = "../support/guid", features = ["random"] }
 uniffi = "0.24.1"
+mockall = "0.11"
```

Comment:
Aaah! this is why the dependency summaries were failing, lets remove this from there, it should only be a test dependency!

This way we don't need any fixups and we can land your PR without having to create a new one 👀

Reply:
okay! thank you